docs/02-app/01-building-your-application/06-configuring/01-typescript.mdx

```diff
@@ -59,7 +49,11 @@ If you miss the prompt, you can enable the plugin manually by:
   height="637"
 />
 
-Now, when editing files, the custom plugin will be enabled. When running `next build`, the custom type checker will be used. Further, we automatically create a VSCode settings file for you to automate this process.
+Now, when editing files, the custom plugin will be enabled. When running `next build`, the custom type checker will be used.
+
+> **Good to know:**
+> 
+> Prior to v13.4.0 VS Code would prompt users to set the TypeScript version when running `npm run dev` for the first time. However, this was removed until such time a better solution is implemented. 
```

Comment:
```suggestion
> Prior to v13.4.0, VS Code would prompt users to enable the plugin when running `npm run dev` for the first time. However, this caused issues for some users and was removed until a better solution is implemented. Please follow the steps above to manually enable the plugin. 
```

I realise "prompt users to set the TypeScript version" is strictly accurate, but the secondary effect is what users need to know here IMO :pray: